apps/sample-app/src/index.tsx

```diff
@@ -0,0 +1,36 @@
+import './index.css';
+
+import { PublicClientApplication } from '@azure/msal-browser';
+import { MsalProvider } from '@azure/msal-react';
+import { FluentProvider, teamsLightTheme } from '@fluentui/react-components';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import App from './App';
+import AuthApp from './AuthApp';
+import { msalConfig } from './components/authConfig';
+const msalInstance = new PublicClientApplication(msalConfig);
```

Comment:
We should construct msalInstance inside of the if block at line 18 so we only initialize this when we're actually going to use it